generate.ml

```diff
@@ -146,11 +146,38 @@ let gen_const typ =
   | _ ->
     Cil.mkCast ~force:false ~e:(Cil.zero ~loc) ~newt:typ
 
+let gen_array_elt (lhost, offset as lval) =
+  match Cil.typeOfLval lval with
+  | TArray (_, e, _, _) ->
+    (* Generate random array index in [0, array_length - 1], just for the last
+       array dimension. *)
```

Comment:
Do we ever have more than one dimension? Is that just something you're planning for the future?

Reply:
No, we don't have more than one dimension, ever. At least at the moment.
I'm not planning to do it right away, but it should not be too hard (?!).

The idea for this comment was to have a reminder that the `NoOffset` case is at last position, hence that we operate on last dimension. But I guess it is misleading. I will remove it altogether if you prefer (or delete the `just` part maybe?).

Reply:
Deleting the entire comment or the `just` part are both fine with me.